### PR TITLE
fix: Use Fetch instead of Axios

### PR DESCRIPTION
**Pull request by zacharyjarnagin**
### 🔗 Linked issue

#113

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Originally, the es-file-input component was using the axios instance defined in nuxt. This caused issues when the component was used in CSM since the axios instance in nuxt had a token on every POST request, causing S3 uploads to fail.

Ideally, we could have just created a new nuxt instance, but axios is not a dependency in es-design-system. To get around this, the component is now using the fetch module. The downside is obtaining upload progress is extremely convoluted, so it has been removed.

Removing the uploadProgress event meant I had to modify EsFileThumbnail so the user can see a normal spinner whenever the percent loaded is zero. The percent loaded for a file will now only be either zero or 100, but the functionality for upload progress can easily be added back in.

### 🥼 Testing

- Manual testing of uploading to S3. I had overmind running and posted to olympus by going to `localhost:8104/api` and creating a file, getting the fileUploadUrl, and adding the fields manually under the `onSubmit` method with the key `additionalFields`. Once I did that, I submitted the file and downloaded it from Olympus

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.
- [x] I have documented testing approach

### CODE REVIEW

**Review by RyanMulready — APPROVED** [review]
Code changes look good to me. Can you log a ticket in `ES-DS` around adding back tracking progress? You can leave it unassigned

**Comment by zacharyjarnagin** [comment]
> Code changes look good to me. Can you log a ticket in `ES-DS` around adding back tracking progress? You can leave it unassigned

Added as #161!

File: es-vue-base/src/lib-components/EsFileThumbnail.vue
Comment (gigiwargin) [inline es-vue-base/src/lib-components/EsFileThumbnail.vue:1]: My only nit would be that the docs for this loading state are now a little funky. By default, the spinner is going. When "see loading state" is hit, it shows the progress circle. Might want to do two?

**Review by gigiwargin — APPROVED** [review]
Works like a charm! Nice job Zach 🥳  One comment inline but doesn't block approval!

File: es-vue-base/src/lib-components/EsFileThumbnail.vue
Comment (gigiwargin) [reply es-vue-base/src/lib-components/EsFileThumbnail.vue:1]: I agree that it's the intended behavior! But if I didn't know there were two loading states I might be confused looking at the docs 🤷‍♀️

Comment (zacharyjarnagin) [reply es-vue-base/src/lib-components/EsFileThumbnail.vue:1]: I changed the button from "Start loading state" to "Start file upload" to try and highlight that the percentage is shown when a file is in the middle of uploading

**Pull request merged**